packages/tests/src/lib/service_node.ts

```diff
@@ -442,6 +474,37 @@ export class ServiceNode {
     );
   }
 
+  async restCall<T>(
+    endpoint: string,
+    method: "GET" | "POST",
+    body: any = null,
+    processResponse: (response: Response) => Promise<T>
+  ): Promise<T> {
+    this.checkProcess();
+
+    return await pRetry(
+      async () => {
+        try {
+          log.info("Making a REST Call: ", endpoint, body);
+          const options: RequestInit = {
+            method,
+            headers: new Headers({ "Content-Type": "application/json" })
+          };
+          if (body) options.body = JSON.stringify(body);
+
+          const response = await fetch(`${this.httpUrl}${endpoint}`, options);
+          log.info(`Received REST Response: `, response.status);
+          return await processResponse(response);
+        } catch (error) {
+          log.error(`${this.httpUrl} failed with error:`, error);
+          await delay(10);
```

Comment:
nit: why the delay here?

Reply:
copied from old function. removed and it doesn't break tests